recipe/build.sh

```diff
@@ -44,8 +44,11 @@ args+=" -D PKG_SRD=ON"
 args+=" -D PKG_USER-VCSGC=ON"
 args+=" -D PKG_VORONOI=ON"
 args+=" -D WITH_GZIP=ON"
+# plumed
+args+=" -D DOWNLOAD_PLUMED=yes"
```

Comment:
To maintain reproducibility, it would be great to add the plumed sources in the `meta.yaml` file, rather than having the LAMMPS setup download them.

Reply:
Sorry didn't meant to download plumed. It was just a wrong copy paste.